client/src/pages/tasks/Show.js

```diff
@@ -293,6 +293,29 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {task.descendantTasks && task.descendantTasks.length > 0 && (
+                    <TaskParentTask
+                      dictProps={Settings.fields.task.childTask}
+                      name="childTask"
+                      component={FieldHelper.ReadonlyField}
+                      humanValue={
+                        task.descendantTasks && task.descendantTasks.length > 0 && (
+                          <>
+                            {task.descendantTasks.map((tasks) => {
+                              console.log(tasks)
+                              return (
+                                <tr>
+                                  <LinkTo modelType="Task" model={tasks}>
```

Comment:
Should probably use the breadcrumb trail here for consistency.

Reply:
I think we do not need Breadcrumb trail in this page for sub tasks. Cause we have the child and we are in their parent's page. If we use it for sub tasks we can see only one more step but its unnecessary